hidden.css

```
@@ -1,13 +1,13 @@
 .visual-refresh {
-    div[class^="base_"] {
-        grid-template-rows: auto;
+  --custom-app-top-bar-height: 0 !important;
 
-        & > div[class^="bar_"] {
-            display: none;
-        }
+  div[class^="base_"] {
```

```suggestion
  div.base_c48ade {
```
I have switched to using classes with hashes due to performance reasons